diff --git a/torchtitan/components/metrics.py b/torchtitan/components/metrics.py
index 084c2c4..9f2ead9 100644
--- a/torchtitan/components/metrics.py
+++ b/torchtitan/components/metrics.py
@@ -144,7 +144,8 @@ class WandBLogger(BaseLogger):
         os.makedirs(log_dir, exist_ok=True)
 
         self.wandb.init(
-            project=os.getenv("WANDB_PROJECT", "torchtitan"),
+            entity=os.getenv("WANDB_ENTITY", "kavn"),
+            project=os.getenv("WANDB_PROJECT", "adaptive-attention"),
             dir=log_dir,
             config=job_config.to_dict(),
         )
